src/stash/src/upgrade/v28_to_v29.rs

```diff
@@ -251,11 +251,6 @@ mod tests {
                 value: Some(objects_v29::database_id::Value::User(42)),
             }),
         };
-        const DANGLING_DATABASE_ID_V29: objects_v29::DatabaseKey = objects_v29::DatabaseKey {
```

Comment:
Why is this going away?

Reply:
It's always been unused, it just wasn't detected until now for some reason. I also double checked with Joe who originally wrote this migration